pkg/network/setup/netstat.go

```diff
@@ -79,40 +79,45 @@ func (c *NetStat) UpdateStatus(vmi *v1.VirtualMachineInstance, domain *api.Domai
 		return nil
 	}
 
-	interfacesStatus, err := c.ifacesStatusFromPodCache(vmi)
-	if err != nil {
-		return err
-	}
-
-	if len(domain.Spec.Devices.Interfaces) > 0 {
-		interfacesStatus = ifacesStatusFromDomain(interfacesStatus, domain.Spec.Devices.Interfaces)
-	}
+	interfacesStatus := ifacesStatusFromDomain(domain.Spec.Devices.Interfaces)
 
+	var err error
 	if len(domain.Status.Interfaces) > 0 {
-		domain.Status.Interfaces = MergeAgentStatusesWithDomainData(domain.Spec.Devices.Interfaces, domain.Status.Interfaces)
+		domain.Status.Interfaces = mergeAgentStatusesWithDomainData(domain.Spec.Devices.Interfaces, domain.Status.Interfaces)
 		interfacesStatus = ifacesStatusFromGuestAgent(interfacesStatus, vmi.Spec.Domain.Devices.Interfaces, domain.Status.Interfaces)
+
+		ifacesSpec := netvmispec.FilterInterfacesSpec(vmi.Spec.Domain.Devices.Interfaces, func(i v1.Interface) bool {
```

Comment:
Instead of `ifacesSpec` I would call it `forwardingIfacesSpec` or maybe `natIfacesSpec`.

Reply:
slirp is NAT-ed
> User Networking is implemented using "slirp", which provides a full TCP/IP stack within QEMU and uses that stack to > implement a virtual NAT'd network.
https://wiki.qemu.org/Documentation/Networking#User_Networking_.28SLIRP.29

Reply:
Hmm.. I meant it is not NAT-ed by kubevirt. The binding itself is not defining a NAT for SLIRP, it treats it as an "application".
For me, it will be confusing, but if you think it is fine, I really do not mind.